addons/web/static/src/views/fields/image/image_field.js

```diff
@@ -22,6 +23,21 @@ function isBinarySize(value) {
     return /^\d+(\.\d*)? [^0-9]+$/.test(value);
 }
 
+/**
+ * Formats a value to be injected in the image's url in order for that url
+ * to be correctly cached and discarded by the browser (the browser caches
+ * fetch requests with the url as key).
+ *
+ * For records, a not-so-bad approximation is to compute that key on the basis
+ * of the record's __last_update field.
+ */
+export function imageCacheKey(value) {
+    if (value instanceof luxon.DateTime) {
+        return serializeDateTime(value).replace(/[^0-9]/g, "");
```

Comment:
You could simply use the timestamp from the luxon object which doesn't require any extra work (`value.ts`)